src/utils/strip-tags.js

```diff
@@ -6,9 +6,7 @@
  * */
 
 function stripTags(string) {
-  const pattern = '<\\w+(\\s+("[^"]*"|\\\'[^\\\']*\'|[^>])+)?>|<\\/\\w+>';
-  const reg = new RegExp(pattern, 'gi');
-  return string.replace(reg, '');
+    return string.replace(/<\/?[a-z][^>]*>/gi, '');
```

Comment:
Can you remove this, I don't think it will take up capitalized tags or some other cases